packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -652,19 +653,30 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     // If autoComplete is on, attempt to find a match from the available options
     if (this.props.autoComplete === 'on') {
       // If autoComplete is on, attempt to find a match where the text of an option starts with the updated value
-      const items = currentOptions
-        .map((item, index) => {
-          return { ...item, index };
-        })
-        .filter(
-          option => option.itemType !== SelectableOptionMenuItemType.Header && option.itemType !== SelectableOptionMenuItemType.Divider
-        )
-        .filter(
+      let items = currentOptions.map((item, index) => {
+        return { ...item, index };
+      });
+
+      if (this.props.autoCompleteFullString === 'on') {
+        items = items.filter(
           option =>
+            option.itemType !== SelectableOptionMenuItemType.Header &&
+            option.itemType !== SelectableOptionMenuItemType.Divider &&
+            this._getPreviewText(option)
+              .toLocaleLowerCase()
+              .includes(updatedValue)
```

Comment:
>updatedValue [](start = 24, length = 12)

What if `updatedValue` is just whitespace, do you want it to match the first string with a space or should it trim leading/trailing whitespace before calling includes?